azurerm/resource_arm_shared_image_version.go

```diff
@@ -282,6 +290,8 @@ func flattenSharedImageVersionTargetRegions(input *[]compute.TargetRegion) []int
 				output["regional_replica_count"] = int(*v.RegionalReplicaCount)
 			}
 
+			output["storage_account_type"] = v.StorageAccountType
```

Comment:
could we cast this to a string here:
```suggestion
			output["storage_account_type"] = string(v.StorageAccountType)
```